starter-code/pizza.js

```diff
@@ -1 +1,167 @@
 // Write your Pizza Builder JavaScript in this file.
+console.log("linked");
+
+
+// var pepperoni;
+// var mushrooms;
+// var greenPeppers;
+// var whiteSauce;
+// var gfCrust;
+//
+// var totalPrice = 10 + pepperoni + mushrooms + greenPeppers + whiteSauce + gfCrust;
+
+
+
+$(document).ready(function() {
+
+//Default page state ------------------
+
+var pepperoni = 0;
+var mushrooms = 1;
+var greenPeppers = 1;
+var whiteSauce = 0;
+var gfCrust = 0;
+
+function getPrice() {
+  var totalPrice = 10 + pepperoni + mushrooms + greenPeppers + whiteSauce + gfCrust;
+
+  $("#final-price").html("$" + totalPrice);
+};
+
+getPrice()
+
+//No pepperoni by default
+$(".pep").hide();
+$(".btn-pepperonni").removeClass("active");
+$("#pep").hide();
+
+//No white sauce by default
+$(".sauce-white").removeClass("sauce-white");
+$(".btn-sauce").removeClass("active");
+$("#sauce-white").hide();
+
+//No gluten free by default
+$(".crust").removeClass("crust-gluten-free");
+$(".btn-crust").removeClass("active");
+$("#gluten-free").hide();
+
+
+//On click on pepperoni, toggle both picture and ingredient in the list
+$(".btn-pepperonni").on("click", function() {
+  $(".pep").toggle();
+  });
+
+$(".btn-mushrooms").on("click", function() {
+  $(".mushroom").toggle();
+  });
+
+$(".btn-green-peppers").on("click", function() {
+  $(".green-pepper").toggle();
+  });
+
+
+//Remove sauce-white
+
+$(".btn-sauce").on("click", function() {
+  if ($(".sauce").hasClass("sauce-white")) {
+    $(".sauce").removeClass("sauce-white")
+  }
+  else {
+    $(".sauce").addClass("sauce-white");
+  }
+  });
+
+
+//Remove gluten free
+
+  $(".btn-crust").on("click", function() {
+    if ($(".crust").hasClass("crust-gluten-free")) {
+      $(".crust").removeClass("crust-gluten-free");
+      $("#gluten-free").hide()
+    }
+    else {
+      $(".crust").addClass("crust-gluten-free");
+      $("#gluten-free").show()
+    }
+    });
+
+
+
+
+// Active Buttons
```

Comment:
Do you see how similar the active buttons event listeners are? Next time you should do one single function and change the different parts by passing params